vita3k/renderer/src/gl/texture.cpp

```diff
@@ -151,6 +154,53 @@ void configure_bound_texture(const renderer::TextureCacheState &state, const Sce
     }
 }
 
+/**
+ * \brief Remove arbitraty blocks from block compressed texture
+ *
+ * \param fmt    Texture base format.
+ * \param dest   Destination texture data. Size must be sufficient enough of ((width + 3) / 4) * ((height + 3) / 4) * 8 or 16 (bytes) depending on texture base format.
+ * \param data   Source data to decompress.
+ * \param width  Texture width.
+ * \param height Texture height.
+ *
+ * \return Void.
+ */
+static void remove_compressed_arbitrary_blocks(SceGxmTextureBaseFormat fmt, void *dest, const void *data, const std::uint32_t width, const std::uint32_t height) {
+    std::size_t w = (width + 3) / 4;
+    std::size_t h = (height + 3) / 4;
+
+    std::size_t a_w = nearest_power_of_two(width);
```

Comment:
Can you rename this function to `next_power_of_two` (nearest is a bit misleading)?

Reply:
Done